testdata/populate_metadata1.bpf.c

```diff
@@ -7,6 +7,11 @@
 
 #define NAME_MAX 255
 
+const volatile int param;
+const volatile int param2;
+
+GADGET_PARAM(param);
```

Comment:
nit: perhaps add a comment indicating that `GADGET_PARAM(param2)` is intentionally omitted to test that param2 is not added to the metadata?